app/models/region.rb

```diff
@@ -19,8 +19,8 @@ class Region < ApplicationRecord
   # A label is a sequence of alphanumeric characters and underscores.
   # (In C locale the characters A-Za-z0-9_ are allowed).
   # Labels must be less than 256 bytes long.
-  def name_to_path_label
-    name.gsub(/\W/, "_").slice(0, MAX_LABEL_LENGTH)
+  def path_label
+    slug.gsub(/\W/, "_").slice(0, MAX_LABEL_LENGTH)
```

Comment:
As we discussed over our catchup, I think this approach is pretty acceptable, and at least a lot better than the alternative of using `name` and then recursively having to manage paths or whatever to avoid name-clashing.

Since Friendly ID maintains uniqueness and hence provides [this backoff](https://github.com/norman/friendly_id/blob/master/lib/friendly_id/slugged.rb#L226) behavior in race-conditions, we should add a uniqueness clause to the path label field as well; because no two paths should ever clash.